web_app/testSite.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Test</title>
+    <link rel="stylesheet" type="text/css" href="src/form.css">
+</head>
+<body>
+    <div id="task-modal" class="modal">
+        <div class="modal-content">
+        <span class="close">&times;</span>
+        <form id="frame">
+            <h1 id="new-task-form-title">Create new Task</h1>
+            <div id="new-task-form-body">
+                <br>
+                <div class="task-parameters">
+                    <div class="parameter-description">Task Name</div>   
+                    <div class="input-item">
+                        <input id="text-input" type="text"></input>
+                    </div>
+                </div>
+                <br>
+                <div class="task-parameters">
+                    <div class="parameter-description">Task description</div>   
+                    <div class="input-item">
+                        <textarea id="task-parameter-description" type="text" rows="5" cols="17"></textarea>
+                    </div>
+                </div>
+                <br>
+                <div class="task-parameters">
+                    <div class="parameter-description">Completion Time</div>
+                    <div class="input-item">
+                        <input id="time" type="time"></input>
+                    </div>
+                </div>
+                <br>
+                <div class="task-parameters">
+                    <div class="parameter-description">Completion Date</div>
+                    <div class="input-item">
+                        <input id="date" type="date"></input>
+                    </div>
+                </div>
+                <br>
+                <div class="task-parameters">
+                    <div class="desccription">Color</div>
+                    <div class="input-item">
+                        <select name="color" id="color">
+                            <option value="1">light</option>
+                            <option value="2">medium</option>
+                            <option value="3">heavy</option>
+                        </select>
+                    </div>
+                </div>
+                <br>
+                <button id="create-task-button" class="close">Create!</button>
```

Comment:
Are this and the actual close button supposed to have the same class?? It seems weird to me that the create button would have the exact same class